cfme/infrastructure/host.py

```diff
@@ -136,7 +140,8 @@ def update(self, updates, validate_credentials=False, from_details=True):
             if validate_credentials:
                 view.endpoints.ipmi.validate_button.click()
         view.flash.assert_no_error()
-        changed = any([changed, credentials_changed, ipmi_credentials_changed])
+        changed = False if cancel else any([changed, credentials_changed,
```

Comment:
I think this buries the intent a bit here, and it would be better to adjust L145 to be:
```python
if changed and not cancel:
```

Reply:
Would be clearer, thanks.